### PR TITLE
Add AbsCaptureTimeExtension and PlayoutDelayExtension

### PR DESCRIPTION
**Pull request by kevmo314**
Both of these are experimental RTP header extensions defined in
libwebrtc.

### CODE REVIEW

File: abscapturetimeextension.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+// EstimatedCaptureClockOffsetDuration produces the estimated time.Duration represented by this extension.
+func (t AbsCaptureTimeExtension) EstimatedCaptureClockOffsetDuration() *time.Duration {
+	if t.EstimatedCaptureClockOffset == nil {
+		return nil
+	}
+	offset := *t.EstimatedCaptureClockOffset
+	duration := time.Duration((offset>>32)&0x7FFFFFFF)*time.Second + time.Duration((offset&0xFFFFFFFF)*1e9/(1<<32))*time.Nanosecond
+	if offset < 0 {
+		duration *= -1
```
Comment (mengelbart) [inline abscapturetimeextension.go:68]: Why absolute and not signed as transmitted?

File: playoutdelayextension.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+// |  ID   | len=2 |       MIN delay       |       MAX delay       |
+// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+type PlayoutDelayExtension struct {
+	minDelay, maxDelay uint16
+}
+
+// Marshal serializes the members to buffer
+func (p PlayoutDelayExtension) Marshal() ([]byte, error) {
+	if p.minDelay >= playoutDelayMaxValue || p.maxDelay >= playoutDelayMaxValue {
```
Comment (mengelbart) [inline playoutdelayextension.go:28]: I think `playoutDelayMaxValue` itself should be allowed, i.e. this line should use `>` instead of `>=`?

Comment (kevmo314) [reply playoutdelayextension.go]: Done, thanks for catching that.

File: abscapturetimeextension.go (hunk 1 above)
Comment (kevmo314) [reply abscapturetimeextension.go]: The prior version was signed as transmitted, just the way I was doing it first stripped the sign. I updated the code to be more clear.

**Review by mengelbart — APPROVED** [review]
Thanks for the updates!

**Pull request merged**